### PR TITLE
People Update MS

### PR DESCRIPTION
**Pull request by bfossen-ce**
People Update MS

### CODE REVIEW

File: data/ms/legislature/Earle-S-Banks-4fa41145-35bf-442b-bdfe-cf54ea53da69.yml [hunk 1]
```diff
@@ -2,30 +2,35 @@ id: ocd-person/4fa41145-35bf-442b-bdfe-cf54ea53da69
 name: Earle S. Banks
 given_name: Earle S.
 family_name: Banks
-email: ebankslaw@aol.com
+email: ebankslaw@aol.com@house.ms.gov
```
Comment (maxknutsen) [inline data/ms/legislature/Earle-S-Banks-4fa41145-35bf-442b-bdfe-cf54ea53da69.yml:5]: @NewAgeAirbender This email looks wrong but I'm not sure whether it should be ebankslaw@aol.com or ebankslaw@house.ms.gov

File: data/ms/legislature/Johnathan-Ray-Lancaster-d0a3f800-01b9-4442-94e2-676c0a364450.yml [hunk 2]
```diff
@@ -1,25 +1,34 @@
 id: ocd-person/d0a3f800-01b9-4442-94e2-676c0a364450
 name: Johnathan Ray Lancaster
+family_name: Lancaster
 email: jlancaster@house.ms.gov
 image: http://billstatus.ls.state.ms.us/members/house/lancaster.jpg
 party:
-- name: Democratic
+- end_date: '2022-12-29'
```
Comment (maxknutsen) [inline data/ms/legislature/Johnathan-Ray-Lancaster-d0a3f800-01b9-4442-94e2-676c0a364450.yml:7]: @NewAgeAirbender This person has an end date of January 2, 2024. In 2021 they switched from being a democrat to republican. So their party should be republican, although it looks like both will be listed here.

File: data/ms/legislature/Juan-Barnett-fcb35ef2-7463-4f8a-9a8a-3bc45b235ecb.yml [hunk 3]
```diff
@@ -2,22 +2,24 @@ id: ocd-person/fcb35ef2-7463-4f8a-9a8a-3bc45b235ecb
...
 given_name: Juan
 family_name: Barnett
-email: jtubb@house.ms.gov@senate.ms.gov
+email: jbarnett@senate.ms.gov
 image: http://billstatus.ls.state.ms.us/members/senate/barnett.jpg
 party:
 - end_date: '2021-12-14'
   name: Democratic
-- name: Republican
+- end_date: '2022-12-29'
```
Comment (maxknutsen) [inline data/ms/legislature/Juan-Barnett-fcb35ef2-7463-4f8a-9a8a-3bc45b235ecb.yml:10]: @NewAgeAirbender This person has always been a democrat as far as I can tell, so their party should be democrat and their end date should be January 2, 2024

File: data/ms/legislature/Kevin-Horan-b339d980-af38-4e76-b53c-215c26191cc1.yml [hunk 4]
```diff
@@ -5,7 +5,9 @@ family_name: Horan
 email: khoran@house.ms.gov
 image: http://billstatus.ls.state.ms.us/members/house/horan.jpg
 party:
-- name: Democratic
+- end_date: '2022-12-29'
```
Comment (maxknutsen) [inline data/ms/legislature/Kevin-Horan-b339d980-af38-4e76-b53c-215c26191cc1.yml:8]: @NewAgeAirbender This person's end date should be january 2, 2024. In 2020 he changed from Democrat to Republican so their party should be republican

File: data/ms/legislature/Shanda-Yates-de5a9a25-1b08-45ba-9aff-e635905498ba.yml [hunk 5]
```diff
@@ -5,21 +5,31 @@ family_name: Yates
 email: syates@house.ms.gov
 image: http://billstatus.ls.state.ms.us/members/house/yates.jpg
 party:
-- name: Democratic
+- end_date: '2022-12-29'
```
Comment (maxknutsen) [inline data/ms/legislature/Shanda-Yates-de5a9a25-1b08-45ba-9aff-e635905498ba.yml:8]: End date should be january 2, 2024. They switched from democrat to independent on January 13, 2022

**Review by maxknutsen — COMMENTED** [review]
@NewAgeAirbender there was 1 weird email address and a few cases where the candidate switching parties resulted in some weird stuff that you will want to review. I left comments on all of them.

File: data/ms/legislature/Earle-S-Banks-4fa41145-35bf-442b-bdfe-cf54ea53da69.yml (hunk 1 above)
Comment (bfossen-ce) [reply data/ms/legislature/Earle-S-Banks-4fa41145-35bf-442b-bdfe-cf54ea53da69.yml:5]: @maxknutsen it should be ebankslaw@aol.com thats whats listed on the MS website on his webpage

**Pull request merged**